tests/foreman/api/test_syncplan.py

```diff
@@ -106,20 +108,21 @@ def test_positive_get_routes(self):
         )
         for response in (response1, response2):
             response.raise_for_status()
-        self.assertEqual(response1.json()['results'], response2.json()['results'])
+        assert response1.json()['results'] == response2.json()['results']
 
 
-class SyncPlanCreateTestCase(APITestCase):
+@pytest.mark.usefixtures("setup_org_for_syncplan")
+class TestSyncPlanCreate:
     """Tests specific to creating new sync plans."""
 
-    @classmethod
-    def setUpClass(cls):
+    @pytest.fixture(scope='class')
+    def setup_org_for_syncplan(self):
         """Create an organization which can be re-used in tests."""
-        super(SyncPlanCreateTestCase, cls).setUpClass()
-        cls.org = entities.Organization().create()
+        TestSyncPlanCreate.org = entities.Organization().create()
```

Comment:
The Org creation happened in every test case, can we put it as a common fixture for all the sync plan test cases?

```
@pytest.fixture(scope='class')
def setup_org_for_syncplan():
    """Create an organization which can be re-used in tests."""
    org = entities.Organization().create()
    yield org
    org.delete()
```

Reply:
Changes made, all tests now use `module_org` fixture